modules/katamari/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
+- Swapped `Optional` to be a class, instead of an object with function fields
+
+### Improved
+- `Optional` objects should now be much faster to construct and work with
+- The `Result` and `Optional` APIs are now thoroughly documented
+- Added private `.tag` and `.value` fields to `Result`, to make them more friendly to view in the debugger or console
```

Comment:
```suggestion
- Swapped `Optional` to be a class, instead of an object with function fields.

### Improved
- `Optional` objects should now be faster to construct and work with.
- The `Result` and `Optional` APIs are now thoroughly documented.
- Added private `.tag` and `.value` properties to `Result`, to make them friendlier to view in the debugger or console.
```
Missing full stops and `much` is very subjective so we shouldn't be including that. In this case we're talking about ~1ms IIRC so I wouldn't call that much faster in the overall scheme of things.